pkg/sync/v5/derive.go

```diff
@@ -126,3 +126,53 @@ func (derivedType) MDSDConfig(cs *api.OpenShiftManagedCluster) (string, error) {
 
 	return string(b.Bytes()), nil
 }
+
+func (derivedType) TDAgentConfig() ([]interface{}, error) {
```

Comment:
why do you need to pull this out into a derived function?

Reply:
The problem was Linting marshal and unmarshal test. But after some playing with indentation looks like it is ok now :/